framerail/src/routes/[slug]/[...extra]/page.svelte

```diff
@@ -257,6 +260,7 @@
     fdata.set("site-id", $page.data.site.site_id)
     fdata.set("page-id", $page.data.page.page_id)
     fdata.set("revision-number", revisionNumber)
+    fdata.set("last-revision-id", $page.data.page_revision.revision_id)
```

Comment:
This sets it as a string? (Assuming since you use `parseInt()` above)

Reply:
All data sent by forms are either `string` or `Blob`, so to get `number`, `boolean` or other types we need to parse it from string.

Reply:
Ah, I see!